src/components/configMaps/ConfigMap.tsx

```diff
@@ -607,6 +609,26 @@ export function ConfigMapForm({
         }
         toggleYamlMode(not)
     }
+
+    const closeDeleteModal = (): void => {
+        setShowDeleteModal(false)
+    }
+
+    const setShowDelete = (): void => {
+        setShowDeleteModal(true)
+    }
+
+    const renderDeleteCIModal = () => {
+        return (
+            <DeleteDialog
+                title={`Delete '${name}' ?`}
+                description={`Are you sure you want to delete this ConfigMap ?`}
```

Comment:
Pass it as direct string instead of wrapping in `{}`

Reply:
Added variable in the description too so please have a look